src/main/java/org/folio/edge/patron/PatronHandler.java

```diff
@@ -256,7 +257,7 @@ protected void handleProxyResponse(RoutingContext ctx, HttpResponse<Buffer> resp
 
   @Override
   protected void handleProxyException(RoutingContext ctx, Throwable t) {
-    logger.error("Exception calling OKAPI", t);
+    logger.error("Exception retrieving data from mod-patron:", t);
```

Comment:
Given that we are changing the logging for both of these cases, please could they be made consistent e.g. use the same language, include the error in the same way